corehq/apps/locations/dbaccessors.py

```diff
@@ -1,13 +1,27 @@
 from corehq.apps.users.models import CommCareUser
 
 
-def get_users_by_location_id(location_id):
-    """
-    Get all users for a given location
-    """
-    return CommCareUser.view(
+def _users_by_location(location_id, include_docs=True, wrap=True):
+    view = CommCareUser.view if wrap else CommCareUser.get_db().view
+    return view(
         'locations/users_by_location_id',
         startkey=[location_id],
         endkey=[location_id, {}],
-        include_docs=True
-    ).all()
+        include_docs=include_docs,
+    )
+
+
+def get_users_by_location_id(location_id, wrap=True):
```

Comment:
I'm open to being told that's not a worthwhile tradeoff; I've just seen `my_ambiguous_return_type_function(wrap=self.wrap)` etc. too many times—what's the return type there!? It encourages passing these options through that are intended to be passed directly to a couchdbkit call, which leads to pretty bad interfaces that are hard to refactor